docker/postgis-tracts/OpenGeo.repo

```diff
@@ -0,0 +1,5 @@
+[opengeo]
```

Comment:
Why this repo instead of standard CentOS and EPEL repos?  Is this more than just postgres and postgis?  Also, looks like this is for CentOS 6, while this image is CentOS 7...and the packages haven't been updated for over a year.
* http://yum.opengeo.org/suite/v4/centos/

Also, I think Postgres 9.4 is supposed to have some significant performance boosts.  May be worth investigating...